flake.nix

```diff
@@ -70,12 +70,12 @@
         zlib.dev
       ] ++ [
 	# Below are overrides to make canokey-qemu library available to qemu built derivative through a qemu override, which qemu is used for other derivatives
-        canokey-qemu # Canokey lib for qemu build-time compilation.
-        (qemu.override {
-          canokeySupport = true; # This override enables Canokey support in QEMU, resulting in -device canokey being available.
-        })
+        #canokey-qemu # Canokey lib for qemu build-time compilation.
+        #(qemu.override {
+        #  canokeySupport = true; # This override enables Canokey support in QEMU, resulting in -device canokey being available.
+        #})
         # Packages for qemu support with Canokey integration from previous override
-        qemu_full #Heavier but contains qemu-img, kvm and everything else needed to do development cycles under docker
+        qemu_full #Heavier but contains qemu-img, kvm and everything else needed to do development/testing cycles under docker
         #qemu # To test make BOARD=qemu-coreboot-* boards and then call make BOARD=qemu-coreboot-* with inject_gpg statement, and then run statement.
         #qemu_kvm # kvm additional support for qemu without all the qemu-img and everything else under qemu_full
```

Comment:
Since qemu_full now includes canokey support, can we remove the commented out canokey-qemu, qemu.override, qemu, and qemu_kvm?

Reply:
Doing.

Reply:
@JonathonHall-Purism fixed with https://github.com/linuxboot/heads/pull/1687/commits/edd4378b60f281c32ff0cf689a40fb1cbc8a0305

Rebuilding and pushing v.0.1.9. I consider this ready for merge?

Reply:
Looks great, thanks for addressing that!